components/site/breadcrumbs.tsx

```diff
@@ -0,0 +1,44 @@
+import { useRouter } from 'next/router';
+import { Breadcrumb } from '@sparkpost/matchbox';
+
+type BreadcrumbProps = {
+  title?: string;
+}
+
+const urlMap: {[key: string]: string} = {
+  'momentum': 'Momentum Documentation',
```

Comment:
Oh this is cool, should we use this in the Algolia dropdown too?

Reply:
Yea we should, that would be nice

Reply:
I guess my only comment here then would be if we do use this for algolia too, we should make this more of a global value so we're updating it once in one place. :-)